### PR TITLE
[CHK-1908] Payment gateway npg

### PR DESCRIPTION
**Pull request by ciuffagianluca**
Edit auth-request policy in order to handle NPG payment gateway client

### Type of changes

- [ ] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### CODE REVIEW

File: src/core/env/dev/terraform.tfvars [hunk 1]
```diff
@@ -168,6 +168,7 @@ ecommerce_ingress_hostname = "weudev.ecommerce.internal.dev.platform.pagopa.it"
 
 ecommerce_xpay_psps_list = "XPAY"
 ecommerce_vpos_psps_list = "BIC36019,PSPtest1,CHARITY_AMEX,CHARITY_IDPSPFNZ,CHARITY_ISP,40000000001,DINERS,BCITITMM,MARIOGAM,73473473473,60000000001,PAYTITM1,POSTE1,ProvaCDI,50000000001,IDPSPFNZ,70000000001,10000000001,idPsp2,irraggiungibile_wisp,prova_ila_1,pspStress50,pspStress79,pspStress80,pspStress81"
+ecommerce_npg_psps_list  = ""
```
Comment (pietro-tota) [inline src/core/env/dev/terraform.tfvars:171]: Shold we remove this modifications into core domain?

File: src/domains/checkout-app/04_apim_checkout.tf [hunk 2]
```diff
@@ -338,6 +338,7 @@ resource "azurerm_api_management_api_operation_policy" "transaction_authorizatio
   xml_content = templatefile("./api/checkout/checkout_ecommerce/v1/_auth_request.xml.tpl", {
     ecommerce_xpay_psps_list = var.ecommerce_xpay_psps_list
     ecommerce_vpos_psps_list = var.ecommerce_vpos_psps_list
+    ecommerce_npg_psps_list  = var.ecommerce_npg_psps_list
```
Comment (pietro-tota) [inline src/domains/checkout-app/04_apim_checkout.tf:341]: Also here, should we need to modify only ecommerce-app domain with `ecommerce/checkout` api path?

File: src/core/env/dev/terraform.tfvars (hunk 1 above)
Comment (ciuffagianluca) [reply src/core/env/dev/terraform.tfvars:171]: You're right!

**Review by pietro-tota — APPROVED** [review]
🚀

File: src/domains/ecommerce-app/env/weu-dev/terraform.tfvars [hunk 3]
```diff
@@ -35,5 +35,6 @@ tls_cert_check_helm = {
 
 ecommerce_xpay_psps_list = "XPAY,IDPSPFNZ,60000000001"
 ecommerce_vpos_psps_list = "BNLIITRR,BIC36019,PSPtest1,CHARITY_AMEX,CHARITY_IDPSPFNZ,CHARITY_ISP,40000000001,DINERS,BCITITMM,MARIOGAM,73473473473,PAYTITM1,POSTE1,ProvaCDI,50000000001,70000000001,10000000001,idPsp2,irraggiungibile_wisp,prova_ila_1,pspStress50,pspStress79,pspStress80,pspStress81"
+ecommerce_npg_psps_list  = ""
```
Comment (infantesimone) [inline src/domains/ecommerce-app/env/weu-dev/terraform.tfvars:38]: we can configure `BCITITMM` for npg in dev env

**Review by infantesimone — APPROVED** [review]

**Pull request merged**